modules/imgproc/test/test_imgwarp_strict.cpp

```diff
@@ -1088,7 +1095,7 @@ void CV_WarpAffine_Test::run_func()
 
 float CV_WarpAffine_Test::get_success_error_level(int _interpolation, int _depth) const
 {
-    return _depth == CV_8U ? 0 : CV_ImageWarpBaseTest::get_success_error_level(_interpolation, _depth);
+    return _depth == CV_8U ? 1.f : CV_ImageWarpBaseTest::get_success_error_level(_interpolation, _depth);
```

Comment:
The change is not relevant after AlgorithmHint introduction. It passes without the change on Intel at least.